Dangerfile

```diff
@@ -0,0 +1,11 @@
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+warn("PR is classified as Work in Progress. When complete, please remove the [WIP] and assign reviewers.") if github.pr_title.include? "[WIP]"
```

Comment:
Does this understand the GitHub Draft/Ready PR system?

Reply:
Also, while this rule is useful, I don't know how much WIP is part of the GitHub PR culture.

Reply:
You might want to add "[wip]", "wip", and "WIP" if you can't do some equivalent of `toUpper()`.

Reply:
As per our slack conversation, WIP branches are sparse and have no definitive standard within the realm of GitHub. Its up to the companies themselves to enforce. As such, I think leaving out this rule is better for now. However, the `toUpper()` comment makes sense and should be used within the rest of the Dangerfile.

Reply:
Removed via: 0eb3fe0b3868536112e78c5f68e0d6617a5e719f